components/automate-ui/src/styles.scss

```diff
@@ -11,11 +11,23 @@
     top: 68px;
   }
 
+  &[hasUserNotifications] {
+    chef-side-panel {
+      max-height: calc(100% - 120px);
```

Comment:
Could you add a comment that indicates what the 120px is?
Same thing for the 162px below. Thx!